packages/flutter/test/material/text_field_test.dart

```diff
@@ -8919,6 +8919,1124 @@ void main() {
     },
   );
 
+  group('Triple tap/click', () {
+    testWidgets(
+      'Can triple tap to select a paragraph on mobile platforms when tapping at a word edge',
+      (WidgetTester tester) async {
+        // TODO(Renzo-Olivares): Enable for iOS, currently broken because selection overlay blocks the tapanddraggesturerecognizer.
+        final TextEditingController controller = TextEditingController();
+        final bool isTargetPlatformApple = defaultTargetPlatform == TargetPlatform.iOS;
+
+        await tester.pumpWidget(
+          MaterialApp(
+            home: Material(
+              child: TextField(
+                dragStartBehavior: DragStartBehavior.down,
+                controller: controller,
+                maxLines: null,
+              ),
+            ),
+          ),
+        );
+
+        const String testValue = 'Now is the time for\n' // 20
```

Comment:
`testValue` seems to be the same for all the tests (well, two variants). Maybe use a single const String for all `group('Triple tap/click')`?  This source file (and its cupertino counterpart) is becoming really huge...